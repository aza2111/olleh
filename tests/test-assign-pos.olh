//test-assign-positive:
//assigns variables of primitive types and prints their values

int x = 0;
print x;
x = -1;
print x;
x = 89;
print x;

char c;
c = 'x';
print c;

boolean b = false;
print b;

String s = "string";
print s;
