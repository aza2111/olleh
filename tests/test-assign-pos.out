0
-1
89
'x'
false
string
